.jest.json

```diff
@@ -1,18 +1,6 @@
 {
-  "collectCoverage": true,
+  "preset": "ts-jest",
```

Comment:
Is this what requires `transform`, `testRegex`, and `moduleFileExtensions` to no longer be necessary?

Reply:
Yup

Reply:
https://huafu.github.io/ts-jest/user/config#jest-preset